README.md

```diff
@@ -20,6 +20,82 @@ You can install Merlin Systems with `pip`:
 pip install merlin-systems
 ```
 
+### Installing Merlin Systems from Source
+
+Merlin Systems can be installed from source by cloning the GitHub repository and running `setup.py`
+
+```bash
+git clone https://github.com/NVIDIA-Merlin/systems.git
+cd systems && python setup.py develop
+```
+
+### Running Merlin Systems from Docker
+
+Merlin Systems is installed on multiple Docker containers, which are available in the [NVIDIA Merlin container repository](https://catalog.ngc.nvidia.com/containers?filters=&orderBy=dateModifiedDESC&query=merlin):
+
+| Container Name             | Container Location | Functionality |
+| -------------------------- | ------------------ | ------------- |
+| merlin-inference           | https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-inference           | Merlin frameworks and Triton Inference Server |
+| merlin-tensorflow-inference            | https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-tensorflow-inference            | Merlin frameworks selected for only Tensorflow support and Triton Inference Server                    |
+
+If you want to add support for GPU-accelerated workflows, you will first need to install the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-docker) to provide GPU support for Docker. You can use the NGC links referenced in the table above to obtain more information about how to launch and run these containers.
+
+## Getting Started with Merlin Systems
```

Comment:
How similar is this to the example notebook? Could we maybe condense it down to a quick example that shows what it looks like to build and export a simple ensemble and refer people to the notebook for details?